src/reanimated2/frameCallback/FrameCallbackRegistryUI.ts

```diff
@@ -1,11 +1,26 @@
 import { runOnUI } from '../core';
 
-export default interface FrameCallbackRegistryUI {
-  frameCallbackRegistry: Map<number, () => void>;
+type CallbackDetails = {
+  callback: (frameInfo: FrameInfo) => void;
+  startTime: number | undefined;
+};
+
+export type FrameInfo = {
+  timestamp: number;
+  duration: number | undefined;
+  timeSinceFirstFrame: number;
+};
+
+interface FrameCallbackRegistryUI {
+  frameCallbackRegistry: Map<number, CallbackDetails>;
   frameCallbackActive: Set<number>;
   isFrameCallbackRunning: boolean;
```

Comment:
Is this property necessary? It looks like we only assign `false` and never use it, but I might be wrong.

Reply:
We do use it, but actually `isFrameCallbackRunning == (frameCallbackActive.size > 0)` is always true, so we could get rid of it I think.